### PR TITLE
Fix sort corrupting graph

### LINKED ISSUE #1732: Sorting the graph breaks it
Whenever I set `graph.do_sort: true` it results in a broken graph. The location index is huge, and all routing requests fail. I traced the issue down to the sort code copying over the old base/adjacent node IDs to the new edge once it gets created since `copyProperties` is copying node IDs from old to new edge. I have a pull request ready with a test that illustrates the failure and a potential fix.

### PR DESCRIPTION
**Pull request by msbarry**
Fix #1732 by making `copyProperties` not copy over the node IDs from old to new edge. No tests fail if I just remove the the call to `writeEdge` entirely in that method, but I'm assuming the reverse handling needs to be in there for a reason.

### CODE REVIEW

**Comment by karussell** [comment]
Oh, this looks indeed ugly. Thanks for the quick fix! 

Please add yourself as [a contributor](https://github.com/graphhopper/graphhopper/blob/master/CONTRIBUTORS.md) and [send us the CLA](https://github.com/graphhopper/graphhopper/blob/master/CONTRIBUTING.md#license-agreement).

File: core/src/test/java/com/graphhopper/util/GHUtilityTest.java [hunk 1]
```diff
@@ -67,6 +68,16 @@ Graph initUnsorted(Graph g) {
         return g;
     }
 
+    double getLengthOfAllEdges(Graph graph) {
+        double distance = 0;
+        DistanceCalc calc = new DistanceCalc2D();
+        AllEdgesIterator iter = graph.getAllEdges();
+        while (iter.next()) {
+            distance += iter.fetchWayGeometry(3).calcDistance(calc);
```
Comment (karussell) [inline core/src/test/java/com/graphhopper/util/GHUtilityTest.java]: Is this also possible with iter.getDistance()?

**Comment by karussell** [comment]
Oh, this looks indeed ugly. Thanks for the quick fix! 

Please add yourself as [a contributor](https://github.com/graphhopper/graphhopper/blob/master/CONTRIBUTORS.md) and [send us the CLA](https://github.com/graphhopper/graphhopper/blob/master/CONTRIBUTING.md#license-agreement).

File: core/src/test/java/com/graphhopper/util/GHUtilityTest.java (hunk 1 above)
Comment (karussell) [reply core/src/test/java/com/graphhopper/util/GHUtilityTest.java]: This does not work then. Will look a bit deeper whats wrong here.

**Comment by msbarry** [comment]
Thanks @karussell! Sent a request for the CLA and added myself as a contributor.  I removed the entire `writeEdge` call and all of the tests still pass, but `getDistance` doesn't seem to verify this failure in the tests so kept as-is and added a comment.

File: core/src/test/java/com/graphhopper/util/GHUtilityTest.java (hunk 1 above)
Comment (msbarry) [reply core/src/test/java/com/graphhopper/util/GHUtilityTest.java]: getDistance doesn't seem to verify this failure in the tests so kept as-is and added a comment.

**Comment by karussell** [comment]
Thanks!

**Pull request merged**